src/Microsoft.Docs.Template/Models/Attributes.cs

```diff
@@ -17,10 +17,26 @@ public class DataSchemaAttribute : Attribute { }
     [AttributeUsage(AttributeTargets.Class)]
     public class PageSchemaAttribute : DataSchemaAttribute { }
 
-    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = true)]
     public abstract class DataTypeAttribute : Attribute
     {
+        private int _order = -1;
+
         public virtual Type TargetType => typeof(string);
+
+        public int Order
```

Comment:
is RangeAttribute applicable here?

Reply:
What is RangeAttribute used for?

---
In reply to: [223546032](https://github.com/dotnet/docfx/pull/3479#discussion_r223546032) [](ancestors = 223546032)

Reply:
The idea is from [https://github.com/dotnet/corefx/blob/f5ba053ec0ff8a3a74917f47a56b2927de27473e/src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Schema/ColumnAttribute.cs#L47](https://github.com/dotnet/corefx/blob/f5ba053ec0ff8a3a74917f47a56b2927de27473e/src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Schema/ColumnAttribute.cs#L47)
We need to sort the attributes based on Order, to apply them accordingly.

---
In reply to: [223546194](https://github.com/dotnet/docfx/pull/3479#discussion_r223546194) [](ancestors = 223546194,223546032)

Reply:
`Order` should be fixed (get only), markdown is always smaller than xrefproperty